specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-07-01-preview/elasticsan.json

```diff
@@ -1040,6 +1146,21 @@
           },
           {
             "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "x-ms-access-soft-deleted-resources",
+            "description": "Optional, used to get soft deleted volumes if set to true. If it is not set for a soft deleted volumes, it will return 404. If set for active volumes, it will return 404.",
```

Comment:
Same, this is list operation, it should not return 404, but should return nothing if not object should be list out.
Please describe list behavior difference of with/without this header.